code/game/objects/items.dm

```diff
@@ -53,6 +52,10 @@
 	var/vending_cat = null// subcategory for vending machines.
 	var/list/dynamic_overlay[0] //For items which need to slightly alter their on-mob appearance while being worn.
 
+	var/is_shrapnel = 0
+	var/shrapnel_amount = 0
+	var/shrapnel_type = ""
```

Comment:
The heck

I have no idea why guns do it the same way too

I figure it's probably fine if BYOND catches up to it anyways